### ISSUE TITLE
Cannot bind more than one function with viewport.changed

### ISSUE THREAD

**Issue by meirlo** [maciej-gurban/responsive-bootstrap-toolkit#14]
Hi
Trying to bind two functions with viewport.changed. 
the second function won't run.

for example:
```javascript
$(window).bind('resize', function() {
            viewport.changed(function(){
                console.log('foo');
            });
        });
$(window).bind('resize', function() {
            viewport.changed(function(){
                console.log('bar');
            });
        });
```
only foo will be printed to console, bar will never show.

Thanks,
Meir

**Comment by maciej-gurban** [comment]
Hi Meir,

Thanks for reporting this problem. This issue has been fixed in the latest, 2.4.2, version. The docs changed slightly, and now to attach `viewport.changed` to window resize listener the following way is advised:

````javascript
$(window).resize(
    viewport.changed(function(){
        if( viewport.is('xs') ) {
            // ...
        }
    })
});
````

**Comment by meirlo** [comment]
Thanks! 
Now it's working :)
